### DISCUSSION TITLE
How to verify email in password in server side

### DISCUSSION THREAD

**Discussion by nivekithan** [appwrite/appwrite#5618] [Q&A] [answered]
Hey, I am working on web application using remix framework. In that application a user can signIn using email and password.

This is easy to implement by using `account.createEmailSession` method from `appwrite` client sdk. But the problem is I am not using `appwrite` sdk instead I am using `node-appwrite` sdk in backend. 

In `node-appwrite` sdk I could not find any method on both `Users` api and `Account` api which allows me to verify `email` and `password` and returns me the `Users` object. 

Please let me know correct way to do this.

PS: I would definitely used `appwrite` sdk if I was working on SPA using vite

**Comment by byawitz** [comment, marked as answer]
In this repo https://github.com/Meldiron/appwrite-ssr-remix you can see an example of to handle login with `remix` & `appwrite`
